index.css

```diff
@@ -91,6 +100,37 @@ body {
   padding-right: 8%;
 }
 
+.connect {
+  font-weight: 500px;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7b86ece6;
+  font-style: normal;
+  padding-left: 10%;
+  margin-top: 2%;
+}
+
+.main-heading {
+  padding-left: 75%;
+  padding-right: 12%;
```

Comment:
- [ ] Good job implementing responsive values in your CSS rules per project requirements. 👍